### ISSUE TITLE
spotless-maven-plugin 2.33.0 spams the Maven log

### ISSUE THREAD

**Issue by jqno** [diffplug/spotless#1582]
- [ ] summary of problem

Since I've upgraded to spotless-maven-plugin 2.33.0, Spotless spams the Maven log with messages about formatting specific files. 2.32.0 didn't do this.

- [ ] gradle or maven version

Maven 3.8.7

- [ ] spotless version

2.33.0 of the spotless-maven-plugin, with prettier 2.5.1 and prettier-java 1.6.1

- [ ] operating system and version

Fedora Linux 37

- [ ] copy-paste your full Spotless configuration block(s), and a link to a public git repo that reproduces the problem if possible

```xml
<plugin>
    <groupId>com.diffplug.spotless</groupId>
    <artifactId>spotless-maven-plugin</artifactId>
    <version>${version.spotless.maven}</version>
    <configuration>
        <formats>
            <format>
                <includes>
                    <include>src/*/java/**/*.java</include>
                </includes>
                <prettier>
                    <devDependencies>
                        <prettier>${version.spotless.prettier}</prettier>
                        <prettier-plugin-java>${version.spotless.prettier-java}</prettier-plugin-java>
                    </devDependencies>
                    <config>
                        <!-- Changes here must be reflected in .prettierrc.yml -->
                        <tabWidth>4</tabWidth>
                        <printWidth>100</printWidth>
                        <trailingComma>none</trailingComma>
                    </config>
                </prettier>
            </format>
        </formats>
    </configuration>
    <executions>
        <execution>
            <goals>
                <goal>check</goal>
            </goals>
        </execution>
    </executions>
</plugin>
```

Public repo: https://github.com/jqno/equalsverifier/

You can reproduce it by running `mvn clean verify` on the repo

- [ ] copy-paste the full content of any console errors emitted by `gradlew spotless[Apply/Check] --stacktrace`

```
[INFO] creating formatter function (starting server)
[INFO] Preparing NodeServerLayout[nodeModulesDir=/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070, packageJsonFile=/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070/package.json, serveJsFile=/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070/serve.js, npmrcFile=/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070/.npmrc] for npm step com.diffplug.spotless.npm.PrettierFormatterStep$State.
[INFO] Prepared NodeServerLayout[nodeModulesDir=/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070, packageJsonFile=/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070/package.json, serveJsFile=/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070/serve.js, npmrcFile=/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070/.npmrc] for npm step com.diffplug.spotless.npm.PrettierFormatterStep$State in 0 ms.
[INFO] running npm install in /home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070 for npm step com.diffplug.spotless.npm.PrettierFormatterStep$State
[INFO] npm install finished in 1057 ms in /home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/target/spotless-prettier-node-modules-ad1e3c72c0bfb53668bed27bc1ffe070 for npm step com.diffplug.spotless.npm.PrettierFormatterStep$State
[INFO] formatting String 'package nl.jqno.equalsverifier;

import edu.umd.cs[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/EqualsVerifierReport.java'
[INFO] formatting String 'package nl.jqno.equalsverifier;

import java.util.[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/Func.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.api;

import nl.jqn[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/api/EqualsVerifierApi.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.api;

import static[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/api/RelaxedEqualsVerifierApi.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.api;

import java.u[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/api/MultipleTypeEqualsVerifierApi.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.api;

import java.u[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/api/SingleTypeEqualsVerifierApi.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/AbstractDelegationChecker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/CachedHashCodeChecker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/Checker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/ExamplesChecker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/FieldInspector.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/HierarchyChecker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/MapEntryHashCodeRequirementChecker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/NullChecker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/RecordChecker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers;
[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/SignatureChecker.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers.f[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/fieldchecks/ArrayFieldCheck.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers.f[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/fieldchecks/FieldCheck.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers.f[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/fieldchecks/FloatAndDoubleFieldCheck.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers.f[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/fieldchecks/MutableStateFieldCheck.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers.f[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/fieldchecks/ReflexivityFieldCheck.java'
[INFO] formatting String 'package nl.jqno.equalsverifier.internal.checkers.f[...]' in file '/home/jqno/w/equalsverifier/equalsverifier/equalsverifier-core/src/main/java/nl/jqno/equalsverifier/internal/checkers/fieldchecks/SymmetryFieldCheck.java'
```

...and a lot more of this.

**Comment by nedtwigg** [comment]
Looks like this is overzealous logging in only the prettier step

https://github.com/diffplug/spotless/blob/74b344313e2b4b5e017048f6737fc33fa38b5657/lib/src/main/java/com/diffplug/spotless/npm/PrettierFormatterStep.java#L123

We should either remove this or downgrade it to debug. I lean towards just remove...

**Comment by simschla** [comment]
agreed, in this case removing the logger would be no loss 😉

**Comment by nedtwigg** [comment]
Fixed in `plugin-gradle 6.16.0` and `plugin-maven 2.34.0`.

**Comment by jqno** [comment]
Thanks, that was fast!
